src/mlpack/core/data/load.cpp

```diff
@@ -0,0 +1,33 @@
+#include "load.hpp"
```

Comment:
Hi, load.hpp already got the license but load_csv.hpp did not. I fixed that part already